### DISCUSSION TITLE
Update for new expansion

### DISCUSSION THREAD

**Discussion by Tyrenias** [xivdev/Penumbra#430] [General]
is there any sort of estimate regarding when penumbra will be updated to work with the new expansion?

**Comment by Theo-Asterio** [comment]
The discord is the better place to watch for when updates occur. Not the github, there's no ETA.

**Comment by reiichi001** [comment]
It's currently a work in progress and not ready for stable/public release. While there is testing going on, users expecting a fully working plugin should wait for the stable release. ....or you're going to run into a bunch of problems and might crash your game.

Estimates are impossible to provide because plugins updates aren't straightforward or predictable. There's a lot of reverse engineering work that goes into making Penumbra function.
